src/Uniphar.Dotnet.Cli/Commands/GetTokenCommand.cs

```diff
@@ -11,23 +11,42 @@ public sealed class Settings : CommandSettings
         public required Guid ClientId { get; set; }
 
         [CommandOption("-s|--scope")]
-        [DefaultValue(".default")]
+        [DefaultValue(null)]
         public string? Scope { get; set; }
+
+        [CommandOption("--useCIAM")]
```

Comment:
Looking 5 minutes ahead to the next time a new authority needs supporting - I'd suggest this could be `--authority ms`/`--authority ciam` (`ms` as default). We might also consider supporting an arbitrary URI so partner teams can test against new authorities before requesting an addition to the list (with the CLI outputting a warning that a URI should only be a temporary situation, and they should send a PR here to add their authority to the list!)